src/Umbraco.Cms.ManagementApi/Controllers/ExamineManagement/RebuildIndexExamineManagementController.cs

```diff
@@ -0,0 +1,120 @@
+﻿using Examine;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Umbraco.Cms.Core.Cache;
+using Umbraco.Cms.Infrastructure.Examine;
+using Umbraco.Cms.ManagementApi.Services;
+
+namespace Umbraco.Cms.ManagementApi.Controllers.ExamineManagement;
+
+[ApiVersion("1.0")]
+public class RebuildIndexExamineManagementController : ExamineManagementControllerBase
+{
+    private readonly IExamineManagerService _examineManagerService;
+    private readonly ILogger<RebuildIndexExamineManagementController> _logger;
+    private readonly IIndexRebuilder _indexRebuilder;
+    private readonly IAppPolicyCache _runtimeCache;
+
+    public RebuildIndexExamineManagementController(
+        IExamineManagerService examineManagerService,
+        ILogger<RebuildIndexExamineManagementController> logger,
+        AppCaches runtimeCache,
+        IIndexRebuilder indexRebuilder)
+    {
+        _examineManagerService = examineManagerService;
+        _logger = logger;
+        _indexRebuilder = indexRebuilder;
+        _runtimeCache = runtimeCache.RuntimeCache;
+    }
+
+    /// <summary>
+    ///     Rebuilds the index
+    /// </summary>
+    /// <param name="indexName"></param>
+    /// <returns></returns>
+    [HttpPost("rebuildIndex")]
+    [MapToApiVersion("1.0")]
+    [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(typeof(OkResult), StatusCodes.Status200OK)]
+    public async Task<IActionResult> RebuildIndex(string indexName)
```

Comment:
I wonder if this endpoint should simply be `Rebuild`? The other endpoints have no "Index" postfix.